pkg/api/vlabs/validate_test.go

```diff
@@ -780,7 +779,7 @@ func TestProperties_ValidateInvalidExtensionProfiles(t *testing.T) {
 		p := getK8sDefaultProperties(true)
 		p.ExtensionProfiles = test.extensionProfiles
 		err := p.Validate(true)
-		if !reflect.DeepEqual(err, test.expectedErr) {
+		if !helpers.EqualError(err, test.expectedErr) {
```

Comment:
@CecileRobertMichon @jackfrancis FYI I changed this and added a new helper method as the reflect Deep Equals is a possible anti-pattern

https://github.com/golang/lint/issues/232